packages/components/color-picker/src/components/alpha-slider.vue

```diff
@@ -109,12 +109,32 @@ export default defineComponent({
       }
     }
 
+    const getClientXY = (event: MouseEvent | TouchEvent) => {
```

Comment:
This method is used in several places. May be is it better to extract it alone.

Reply:
You are right, I added `getClientXY` as a util.